server-code/Projects.js

```diff
@@ -25,7 +25,7 @@ function getAllProjects() {
  * @return {String} – The corresponding HTML hyperlink
  */
 function getHTMLLink(url, displayText) {
-    var html = `<a href="` + url + `">` + displayText + `</a>` 
+    var html = `<a href="` + url + `"` + `target="_blank" rel="noopener noreferrer"` + `>` + displayText + `</a>` 
```

Comment:
i just did some separation bc i thought it would be clearer but i could also change this to 
`<a href="` + url + `"target="_blank" rel="noopener noreferrer">` + displayText + `</a>` 
which one do you prefer

Reply:
I would be happy with both, but maybe the version with fewer string concatenations is slightly cleaner. However, the cleanest method would be to make use of the substitution feature backticks provide to insert the variables like so:

``var html = `<a href="${url}" target="_blank" rel="noopener noreferrer"> ${displayText} </a>`;``

Reply:
Updated